package.json

```diff
@@ -255,6 +255,10 @@
     "yaserver": "^0.4.0"
   },
   "resolutions": {
+    "semver": "^7.5.4",
+    "micromatch": "^4.0.8",
+    "braces": "^3.0.3",
+    "ws": "^7.5.10",
```

Comment:
I checked out this branch and ran `yarn`, which resulted in some `yarn.lock` changes I think we'll want to commit.